### ISSUE TITLE
MRAN seems to be down

### ISSUE THREAD

**Issue by Civis-agarcia** [Microsoft/microsoft-r-open#107]
Tried to access the package yesterday, at 2019-11-10 02:10:31 AM, but met with following error in log:

```
Warning: unable to access index for repository https://mran.microsoft.com/snapshot/2019-07-05/src/contrib: 
2019-11-10 02:10:31 AM
cannot open URL 'https://mran.microsoft.com/snapshot/2019-07-05/src/contrib/PACKAGES'
```

Has anybody else had problems accessing this package?

**Comment by mehul16** [comment]
The Microsoft CRAN time machine seems to be down. Currently, unable to install any package.
23-12-2019 02:42 IST.

`Warning: unable to access index for repository https://mran.microsoft.com/snapshot/2019-04-15/src/contrib:
  cannot open URL 'https://mran.microsoft.com/snapshot/2019-04-15/src/contrib/PACKAGES'`

**Comment by marcos-garcia** [comment]
Hi, I'm trying to access one of your snapshots, but it seems to be down again. This is the warning that I get:

```
`Warning: unable to access index for repository https://mran.microsoft.com/snapshot/2019-07-05/src/contrib:
cannot open URL 'https://mran.microsoft.com/snapshot/2019-07-05/src/contrib/PACKAGES'`
```

**Comment by andredmoliveira** [comment]
Facing the same issue when trying to access a MRAN snapshot :
```
(converted from warning) unable to access index for repository https://mran.microsoft.com/snapshot/2019-12-12/src/contrib: 
cannot open URL 'https://mran.microsoft.com/snapshot/2019-12-12/src/contrib/PACKAGES'
```

**Comment by harlecin** [comment]
Same issue here
```
Warning: unable to access index for repository https://mran.microsoft.com/snapshot/2019-04-15/src/contrib:
```

PS: thx a lot for providing MRAN! It is seriously awesome:)

**Comment by rsmith54** [comment]
yes, it's down again :( any updates?

**Comment by vonGameTheory** [comment]
Yep -- MRAN down since at least yesterday

**Comment by rsmith54** [comment]
working again for us, but is this the correct location to post these issues for the future?

**Comment by vonGameTheory** [comment]
Yeah, starting working again.  I'm not sure there is a correct location since Microsoft seems to have abandoned R Open.  Gonna have to switch to regular R distro soon...

**Comment by JeffHCross** [comment]
I seem to be getting this error on my Linux distribution but not on Windows. I haven't had any issues installing packages from MRAN, just using setSnapshot or checkpoint(). Any suggestions?
